src/app/tests/suites/certification/Test_TC_CC_2_1.yaml

```diff
@@ -28,33 +28,35 @@ tests:
               - name: "nodeId"
                 value: nodeId
 
-    - label: "Validate constraints of attribute: CurrentHue"
-      PICS: CC.S.A0000
+    - label: "TH reads CurrentHue attribute from DUT"
+      PICS: CC.S.A0000 && CC.S.F00
       command: "readAttribute"
       attribute: "CurrentHue"
       response:
           value: 0
```

Comment:
Shouldn't be checking defaults. if a functional test is run before this, it will never be at the default value.
Same applies to all CC attributes.

Reply:
@yinyihu-silabs thanks then i will remove all the default value checks for test TC-CC-2.1

Reply:
@yinyihu-silabs rebased the PR can you please verify TC-CC-2.1